cmd/octorpki/octorpki.go

```diff
@@ -950,7 +955,9 @@ func (s *state) ServeROAs(w http.ResponseWriter, r *http.Request) {
 			w.Header().Set("Cache-Control", fmt.Sprintf("max-age=%v", maxAge))
 		}
 
+		s.ROAListLock.RLock()
 		tmp := s.ROAList
```

Comment:
is `s.ROAList` still usable when it is being updated?
This line won't copy the slice, just a reference to it.